src/react-contenteditable.js

```diff
@@ -7,20 +7,18 @@ export default class ContentEditable extends React.Component {
   }
 
   render() {
-    var props = {};
-    Object.assign(props, this.props);
-    delete props.tagName;
-    delete props.html;
```

Comment:
tagName and html were explicitly deleted, now with this PR, they would be transmitted to the child element.